addons/camera/stringtable.xml

```diff
@@ -3,30 +3,39 @@
     <Package name="Camera">
         <Key ID="STR_ZEN_Camera_Camera">
             <English>Camera</English>
+            <Russian>Камера</Russian>
         </Key>
         <Key ID="STR_ZEN_Camera_FollowTerrain">
             <English>Follow Terrain</English>
+            <Russian>Следование Рельефу</Russian>
         </Key>
         <Key ID="STR_ZEN_Camera_FollowTerrain_Description">
             <English>Controls whether the camera's height should adjust with changes in terrain elevation.</English>
+            <Russian>Контролирует должна ли высота камеры регулироваться при изменении высоты местности</Russian>
         </Key>
         <Key ID="STR_ZEN_Camera_AdaptiveSpeed">
             <English>Adaptive Speed</English>
+            <Russian>Адаптивная Скорость</Russian>
         </Key>
         <Key ID="STR_ZEN_Camera_AdaptiveSpeed_Description">
             <English>Adapts the camera's speed based on its current height. Lower height - slower camera movement.</English>
+            <Russian>Адаптирует скорость камеры в зависимости от ее текущей высоты. Чем ниже высота - тем медленее движение камеры.</Russian>
         </Key>
         <Key ID="STR_ZEN_Camera_DefaultSpeedCoef">
             <English>Default Speed Coefficient</English>
+            <Russian>Коэффициент Скорости По Умолчанию</Russian>
         </Key>
         <Key ID="STR_ZEN_Camera_DefaultSpeedCoef_Description">
             <English>Coefficient for the default camera movement speed, higher is faster.</English>
+            <Russian>Коэффициент для скорости движения камеры по умолчанию. Чем выше тем быстрее.</Russian>
         </Key>
         <Key ID="STR_ZEN_Camera_FastSpeedCoef">
             <English>Fast Speed Coefficient</English>
+            <Russian>Коэффициент Быстрой Скорости</Russian>
```

Comment:
I am not sure what this means. Literal translation is ok.

Reply:
When you hold down shift (I think?) in Zeus your camera moves faster. This is the coefficient defining how much faster the camera moves if you hold the key